support/project_readme.md

```diff
@@ -76,6 +76,24 @@ bundle install
 * [App Status Page](http://app.<applicationname>.com/_ping) will give you information about what is running.
 * Alternatively, you can ssh in and check that the application server and web server are both running.
 
+# Testing Strategy
+## Testing approach
+### End User tests
+Due to the nature of this application, End User tests are...
```

Comment:
Should this include a line like: "To run the End User tests: bundle exec spinach"

Reply:
This seems covered by the other section